evidence-based-complementary-and-alternative-medicine.csl

```diff
@@ -0,0 +1,299 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" default-locale="US English" demote-non-dropping-particle="sort-only">
```

Comment:
@KMDjustin, can you change `default-locale="US English"` to `default-locale="en-US"`? (it has to be a proper locale code)